### PR TITLE
Set registry replicas = 1 when no storage specified.

### PR DESCRIPTION
**Pull request by abutcher**

### CODE REVIEW

File: roles/openshift_hosted/tasks/registry/registry.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   when: openshift.hosted.registry.replicas | default(none) is none
 
 - set_fact:
-    replicas: "{{ openshift.hosted.registry.replicas | default((openshift_hosted_registry_nodes_json.stdout | from_json)['items'] | length) }}"
+    replicas: "{{ openshift.hosted.registry.replicas | default(((openshift_hosted_registry_nodes_json.stdout | from_json)['items'] | length) if openshift.hosted.registry.storage.kind | default(none) != none else 1) }}"
```
Comment (detiber) [inline roles/openshift_hosted/tasks/registry/registry.yml]: Shouldn't the test be `is not none`?

Comment (abutcher) [inline roles/openshift_hosted/tasks/registry/registry.yml]: Bumpdated

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
@sdodson ptal

**Comment by sdodson** [comment]
LGTM

**Pull request merged**